lib/facter/nginx_version.rb

```diff
@@ -2,7 +2,7 @@
 
 Facter.add(:nginx_version) do
   confine { Facter.value(:kernel) != 'windows' }
-  confine { Facter.value(:operatingsystem) != 'nexus' }
+  confine { Facter.value(:os)['name'] != 'nexus' }
```

Comment:
I think you can use
```suggestion
  confine { Facter.value('os'.name') != 'nexus' }
```

While it's probably safe to assume `os` is present and there's probably no (measurable) performance difference, but , it's still a good practice to learn.